.github/workflows/code_coverage.yml

```diff
@@ -38,13 +38,7 @@ jobs:
       - name: Install grcov
         run: if [[ ! -e ~/.cargo/bin/grcov ]]; then cargo install grcov; fi
       - name: Test
-        # WARNING: this is not testing the following features: test-esplora, test-hardware-signer, async-interface
-        # This is because some of our features are mutually exclusive, and generating various reports and
-        # merging them doesn't seem to be working very well.
-        # For more info, see:
-        # - https://github.com/bitcoindevkit/bdk/issues/696
-        # - https://github.com/bitcoindevkit/bdk/pull/748#issuecomment-1242721040
-        run: cargo test --features all-keys,compact_filters,compiler,key-value-db,sqlite,sqlite-bundled,test-electrum,test-rpc,verify
+        run: cargo test --all-features
```

Comment:
If you want to test with the feature hardware-signer, you need to install hwi and start the hardware simulator. See the last commit of https://github.com/bitcoindevkit/bdk/pull/873

Reply:
Ah I see so we didn't test with it before. I'm not sure I want to fix that but I also don't want to go back to feature listing. What do you think about commenting this test out for now and doing #872 before `v1.0.0`?

Reply:
(I merged your fix so let's see how far that gets us).